### DISCUSSION TITLE
Can't connect to Github Codespaces, error message is empty

### DISCUSSION THREAD

**Discussion by BasVerlooy** [community/community#40218] [Codespaces]
### Select Topic Area

Bug

### Body

Hi all, I'm experiencing a weird issue with 1 of my codespaces. When I try to connect to it will give an error with no meaningful message: 
<img width="508" alt="image" src="https://user-images.githubusercontent.com/33267625/204155656-cc37ef5e-6f4f-4ef1-9bfe-c53da75f2923.png">
This issue first appeared on Saturday and my first assumption was that Codespaces was down, but the codespaces of my organisation still work just like another personal Codespace. I can't pinpoint the exact moment my codespaces failed to load again, but starting up a new codespace doesn't work, just like deleting the old ones. 

Unfortunately the error message isn't that helpful, but does anyone have a clue what my options are?

**Comment by ossanna16** [comment]
Hi there @BasVerlooy and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by GideonCheruiyot** [comment]
Hello @BasVerlooy, could you please provide your codespace's name, for example the url on your codespace's tab to help investigate?
